src/utils/vote-presets.js

```diff
@@ -21,22 +26,25 @@ const prepareInflationProposal = async (receiverAddress, amount) => {
     .multipliedBy(10 ** ethPNTDecimals)
     .toFixed()
 
-  const withdrawInflationParams = [settings.contracts.financeVault, rawAmount]
-
-  const transferParams = [ethPNTAddress, receiverAddress, rawAmount]
-
-  return [
-    {
-      to: ethPNTAddress,
-      calldata: EthPNT.encodeFunctionData('withdrawInflation', withdrawInflationParams)
-    },
-    {
-      to: settings.contracts.financeVault,
-      calldata: Vault.encodeFunctionData('transfer', transferParams)
-    }
-  ]
+  return {
+    Vault: Vault,
+    EthPNT: EthPNT,
+    rawAmount: rawAmount,
+    ethPNTAddress: ethPNTAddress
+  }
 }
 
+const prepareInflationProposal = async (Vault, EthPNT, ethPNTAddress, receiverAddress, rawAmount) => [
```

Comment:
Pay attention here, `Vault` and `EthPNT` here are parameters overriding the previously defined constants. This should be avoided. Use directly the constant within the function, or assign the parameters other names.

Reply:
Can you clarify this? Which constants are they overriding?

Reply:
@oliviera9 I thought that too, but apparently they are not declared in the outer scope. @envin3 I would rename those without the first capital letter as they could be misleading by the syntax colouring...

Reply:
I've been misled by the definitions at lines 15 and 16 in `prepareInfaltionData()`. So actually you are not overriding anything, the initial capital letter was confusing. You may consider moving the interfaces instantiation inside this function.

Reply:
I didn't instantiate the interface inside the function to reuse the code in all inflation related functions. If I did, I'd have to reinstantiate them also outside the function for the lastly added proposal.

Reply:
Is that bad declaring them at the module level? So they'll be loaded once and available everywhere